client/coniksclient/README.md

```diff
@@ -0,0 +1,35 @@
+# CLI CONIKS Client implementation in Golang
+
+## Usage
+```
+⇒  go install github.com/coniks-sys/coniks-go/client/coniksclient
+⇒  coniksclient -h
+________  _______  __    _  ___  ___   _  _______
+|       ||       ||  |  | ||   ||   | | ||       |
+|       ||   _   ||   |_| ||   ||   |_| ||  _____|
+|       ||  | |  ||       ||   ||      _|| |_____
+|      _||  |_|  ||  _    ||   ||     |_ |_____  |
+|     |_ |       || | |   ||   ||    _  | _____| |
+|_______||_______||_|  |__||___||___| |_||_______|
+
+Usage:
+  coniksclient [command]
+
+Available Commands:
+  init        Creates a config file for the client.
+  lookup      Lookup a name.
+  register    Register a name-to-key binding.
```

Comment:
What we have here is fine for now.

Might be fun to use,
https://godoc.org/golang.org/x/crypto/ssh/terminal